### ISSUE TITLE
backward in loss function

### ISSUE THREAD

**Issue by mohsen0311** [Spandan-Madan/A-Collection-of-important-tasks-in-pytorch#4]
in every iteration in training we call loss.backward . is this inherited from nn.modules  or we should implement it. what i want to do is edulican distance between to vector size of 6 is there any need to implement backward function .
